core/src/main/java/hudson/Functions.java

```diff
@@ -2289,12 +2289,17 @@ public static Icon tryGetIcon(String iconGuess) {
             return null;
         }
 
-        StaplerRequest currentRequest = Stapler.getCurrentRequest();
-        currentRequest.getWebApp().getDispatchValidator().allowDispatch(currentRequest, Stapler.getCurrentResponse());
```

Comment:
I don't know what this code does.
But I think this code isn't necessary and removed it to make it easy to run tests.